opr/assetList.go

```diff
@@ -17,9 +23,80 @@ func (o OraclePriceRecordAssetList) Contains(list []string) bool {
 			return false
 		}
 	}
+
 	return true
 }
 
+// Exchange tells us how much we need to spend given the amount we want is fixed.
+//	?? FROM -> X TO
+//
+//   X TO         to_usd               1
+//  ------    *  --------- = USD * -------- = FROM
+//     1            1               from_usd
+//
+func (o OraclePriceRecordAssetList) ExchangeTo(from string, to string, want int64) (int64, error) {
```

Comment:
both this and exchangefrom seem to only operate on whole numbers, which seems kinda useless. is this supposed to only work for the smallest denomination, aka cents / sats / whatever? if so, it really needs to be pointed out

also the names for ExchangeTo and ExchangeFrom are really confusing. took me a long time to figure out what they're supposed to do. they should be swapped around

i'm not sure we even need the function for "how much X do i need to get Y?", since that's literally just swapping the order of the arguments